src/internal/__tests__/analytics-metadata-test-utils.ts

```diff
@@ -25,7 +25,8 @@ const validateLabels = (labelSelectors: Array<string>, labelsClassNames: Record<
     const parts = current
       .split(/\s+/)
       .filter(part => part.startsWith('.'))
-      .map(part => part.replace('.', ''));
+      .map(part => part.replace('.', ''))
+      .map(part => part.replace(/\[.*\]/, ''));
```

Comment:
why did we add this regex?

Reply:
This test ensures that all classNames used in these objects come from specific files that we will not change for stability.

In this component, the selectors for the group label contain a `.className[data-group-index="n"] .another-className`. We need to remove all attribute-based CSS selectors to be able to run this test.